src/wallet/receive.cpp

```diff
@@ -290,6 +290,15 @@ bool CachedTxIsTrusted(const CWallet& wallet, const CWalletTx& wtx)
     return CachedTxIsTrusted(wallet, wtx, trusted_parents);
 }
 
+Balance GetFullBalance(const CWallet& wallet, const int min_depth, bool avoid_reuse) {
```

Comment:
This doesn't need to be a separate function, just have `GetBalance` include `m_mine_used`. It's up to the caller to figure out what to do with that information.

Reply:
Fixed: updated `GetBalance` to include `m_mine_used`